system-tests/helpers/defaults.py

```diff
@@ -0,0 +1,24 @@
+"""
+Default values for system tests.
+"""
+
+# 1 RIA is 10^9 nRIA
+BASE_AMOUNT = 1000000000
+# RIA is 10^9, WEI is 10^18, 10^9 * 10^9 = 10^18
+ROLLUP_MULTIPLIER = 1000000000
+# 10 RIA
+TRANSFER_AMOUNT = 10
+
+# Sequencer Defaults
+#####################
+SEQUENCER_BRIDGE_ADDRESS = "astria13ahqz4pjqfmynk9ylrqv4fwe4957x2p0h5782u"
+SEQUENCER_BRIDGE_P_KEY = "dfa7108e38ab71f89f356c72afc38600d5758f11a8c337164713e4471411d2e0"
+SEQUENCER_FUNDS_P_KEY = "934ab488f9e1900f6a08f50605ce1409ca9d95ebdc400dafc2e8a4306419fd52"
```

Comment:
I'd use this file only for consts which are shared by multiple other files.  So these three would be better placed in astria_cli.py IMO.  This maybe applies to some other consts in this file?

I'd also recommend renaming the private key variables to use e.g. `...SIGNING_KEY` or `_SECRET_KEY` since `P` could easily be mistaken to mean "public"